src/bitcoin-cli.cpp

```diff
@@ -1027,7 +1053,31 @@ static void ParseGetInfoResult(UniValue& result)
         }
         result_string += "\n";
     }
-
+    
+    result_string += strprintf("Feerate Distribution: \n");
+    
+    std::string dist ("feerate_dist");
+    std::string dist_n;
+    std::string fee_rate;
+    for (int i = 0; i < 3; ++i) {
+        dist_n = dist + std::to_string(i);
+        const double fee_size{result["feerate_dist1"].get_real()};
+        std::string size_bar;
+        GetFeerateDist(fee_size, size_bar);
+        size_bar += " ";
+        
+    switch (i)
+    {    
+    case 0: fee_rate = "1-9"; 
+        break;
+    case 1: fee_rate = "10-19";
+        break;
+    case 2: fee_rate = "20-29";
```

Comment:
These fee rate ranges seem arbitrary. Relevant real fee rates may evolve to be much higher or lower.

Reply:
I used them to test. Was planning to add more once results are as expected. We can do one of these to improve:

1. Add more groups: https://i.imgur.com/ffyusBA.png
2. Define a MIN and MAX, MIN will be 1 and MAX can be highest fee rate used by transactions in mempool. And then divide this in two few groups.

Reply:
Used different groups in last commit: 1-9, 10-99, 100-199